### PR TITLE
Updating guidlines for Platforms (CMS). 

### PR DESCRIPTION
**Pull request by iortz-ethology**
Adding intro information for CMS's and guidlines for when we use WordPress

### CODE REVIEW

File: docs/Platforms.md [hunk 1]
```diff
@@ -114,8 +114,28 @@ Sometimes these overhead costs, and headaches, cut into all the time saved in wr
 
 ## CMS
 
+On today's Internet, content is king. You want to be the top search result on your favorite search engine, do you want to engage with your clients, do you want to expose your brand...Content! Bill Gates said "Content is where I expect much of the real money will be made on the Internet, just as it was in broadcasting."
```
Comment (swilliams) [inline docs/Platforms.md]: Add a source for the BillG quote.

File: docs/Platforms.md [hunk 2]
```diff
@@ -114,8 +114,28 @@ Sometimes these overhead costs, and headaches, cut into all the time saved in wr
...
+
+WordPress platforms allows us to develop custom plugins and themes for our clients to further drive the experience our clients want to provide their clients. Slowly WordPress is evolving to provide a better framework for development teams to create these customs experiences. Because developing custom plugins and themes for WordPress is not as straight forward as it could be, the community became saturated with plugins and themes that are very costly.
+
+Because of the potential complexity and cost that involves providing a custom and unique experience for our clients clients, we focus on using WordPress on simpler experience. When all our clients care is to deliver content, and all they want is for their clients and potential users to be able to read content online, we use WordPress.
+
+However as it stands, many times our clients want to not only provide a platform for their users to read their content, but they want to provide a platform where they can interact with their clients, in comes a more evolve platform **Drupal**.
```
Comment (swilliams) [inline docs/Platforms.md]: "However as it stands" can be stripped out.

File: docs/Platforms.md [hunk 3]
```diff
@@ -114,8 +114,28 @@ Sometimes these overhead costs, and headaches, cut into all the time saved in wr
...
+
+The best part about WordPress is that it’s easy to use and flexible enough for just about anything. That’s the main reason why WordPress has grown so much in popularity.
+
+One of the best features of WordPress is that it is out of the box free. That means you can download WordPress and with little to no development knowledge create a website for your brand to deliver content.
+
+WordPress platforms allows us to develop custom plugins and themes for our clients to further drive the experience our clients want to provide their clients. Slowly WordPress is evolving to provide a better framework for development teams to create these customs experiences. Because developing custom plugins and themes for WordPress is not as straight forward as it could be, the community became saturated with plugins and themes that are very costly.
+
+Because of the potential complexity and cost that involves providing a custom and unique experience for our clients clients, we focus on using WordPress on simpler experience. When all our clients care is to deliver content, and all they want is for their clients and potential users to be able to read content online, we use WordPress.
```
Comment (swilliams) [inline docs/Platforms.md]: First sentence ends a little awkwardly. "we focus on using WordPress on simpler experience." Not sure what that means.

File: docs/Platforms.md (hunk 1 above)
Comment (krantzinator) [reply docs/Platforms.md]: make all three pieces of that second sentence match; you could remove the two `do`s (replacing the second one with `and`), or add a `Do` to the beginning.

**Comment by swilliams** [comment]
Created an Issue (#23) to track against.

File: docs/Platforms.md [hunk 4]
```diff
@@ -114,8 +114,28 @@ Sometimes these overhead costs, and headaches, cut into all the time saved in wr
...
+
+However as it stands, many times our clients want to not only provide a platform for their users to read their content, but they want to provide a platform where they can interact with their clients, in comes a more evolve platform **Drupal**.
+
```
Comment (mmislam101) [inline docs/Platforms.md]: Can we list some 
- Common Wordpress PaaS we'd use to help our clients.
- Plugins and libraries that are useful for bootstrapping development.

Comment (swilliams) [reply docs/Platforms.md]: Common WP PaaS that I believe are good:

* [Pantheon](https://pantheon.io)
* [WPEngine](https://wpengine.com)

**Comment by swilliams** [comment]
I've made updates based on the feedback here. @iortz-ethology @ruulito please review.

**Comment by swilliams** [comment]
@krantzinator @mmislam101 This section is good to go, please review again. ❤️

**Pull request merged**